.github/workflows/qase-patch-validation-create.yaml

```diff
@@ -0,0 +1,33 @@
+name: Qase Patch Validation Create
```

Comment:
Create Patch Validation Test run as this is specific to patch validation. For minor we will need to create a separate test run